processor/cumulativetodeltaprocessor/internal/tracking/tracker.go

```diff
@@ -85,8 +127,27 @@ func (t *MetricTracker) Convert(in MetricPoint) (out DeltaValue, valid bool) {
 		PrevPoint: metricPoint,
 	})
 	if !ok {
+		switch metricID.MetricType {
+		case pmetric.MetricTypeHistogram:
+			*out.HistogramValue = metricPoint.HistogramValue.Clone()
+		case pmetric.MetricTypeSum:
+			out.IntValue = metricPoint.IntValue
+			out.FloatValue = metricPoint.FloatValue
+		}
+		switch t.initialValue {
+		case InitialValueAuto:
+			if metricID.StartTimestamp < t.startTime || metricPoint.ObservedTimestamp == metricID.StartTimestamp {
+				return
+			}
+			out.StartTimestamp = metricID.StartTimestamp
+			valid = true
+		case InitialValueKeep:
+			valid = true
+		case InitialValueDrop:
```

Comment:
nit: we can omit this case.

Reply:
removed, i originally left it in for completeness